dyno-jedis/src/test/java/com/netflix/dyno/jedis/utils/EmbeddedRedisInitializer.java

```diff
@@ -0,0 +1,43 @@
+package com.netflix.dyno.jedis.utils;
+
+import io.netty.channel.ChannelInitializer;
+import io.netty.channel.ChannelPipeline;
+import io.netty.channel.socket.SocketChannel;
+import io.netty.handler.codec.string.StringDecoder;
+import io.netty.handler.codec.string.StringEncoder;
+import io.netty.handler.ssl.SslHandler;
+
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLEngine;
+
+public class EmbeddedRedisInitializer extends ChannelInitializer<SocketChannel> {
+    private final SSLContext sslContext;
+    private final boolean useSsl;
+    private final String response;
+
+    public EmbeddedRedisInitializer(final SSLContext sslContext, final boolean useSsl, final String response) {
+        this.sslContext = sslContext;
+        this.useSsl = useSsl;
+        this.response = response;
+    }
+
+    @Override
+    protected void initChannel(final SocketChannel ch) throws Exception {
+        final ChannelPipeline pipeline = ch.pipeline();
+
+        if (useSsl) {
+            final SSLEngine sslEngine = sslContext.createSSLEngine();
```

Comment:
While setting up the server & client not a part of this library or PR, it would nice to set the integration test server and client up to only use TLS 1.2.

Reply:
Both client and server uses 
```
final SSLContext sslContext = SSLContext.getInstance("TLSv1.2");
```

But one tests failed - can you access this file from failed build  ?
```
file:///home/travis/build/Netflix/dyno/dyno-core/build/reports/tests/test/index.html
```
or at least run build again ? I am pretty sure that this failed randomly, locally it pass.